dm/relay/binlog_writer.go

```diff
@@ -19,30 +19,27 @@ import (
 	"os"
 	"sync"
 
+	"github.com/pingcap/errors"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
 
-	"github.com/pingcap/ticdc/dm/pkg/binlog/common"
 	"github.com/pingcap/ticdc/dm/pkg/log"
 	"github.com/pingcap/ticdc/dm/pkg/terror"
 )
 
 // BinlogWriter is a binlog event writer which writes binlog events to a file.
 type BinlogWriter struct {
-	cfg *BinlogWriterConfig
+	mu sync.RWMutex
```

Comment:
seems `FileWriter.out` is only used in one goroutine and driven by upstream binlog event, so we even need no lock.

Reply:
"filename/offset" state'll be used in binlog reader later.